opm/input/eclipse/Schedule/ScheduleState.hpp

```diff
@@ -43,6 +37,12 @@
 #include <opm/input/eclipse/Schedule/VFPInjTable.hpp>
 #include <opm/input/eclipse/Schedule/RSTConfig.hpp>
 
+#include <chrono>
+#include <cstddef>
+#include <memory>
+#include <optional>
+#include <unordered_map>
+#include <utility>
```

Comment:
Since you're moving this list of headers anyway, maybe we could add `<type_traits>` too, to bring in a declaration of `std::is_same_v<>`?

Reply:
sure.